lib/engine/game/base.rb

```diff
@@ -772,6 +775,18 @@ def operating_round(round_num)
         Round::Operating.new(@corporations, game: self, round_num: round_num)
       end
 
+      def event_close_companies!
+        @log << '-- Event: Private companies close --'
+
+        @companies.each do |company|
+          ability = company.abilities(:close)
```

Comment:
let's use block style 

company.abilities(:close) do |ability|
  next if ability.when == 'never' # note this is a string and not a symbol like you have it here

i think there's a bug with the when stuff that we need to fix i guess when should always be a string?  since phase names are strings